libraries/AP_InertialSensor/BatchSampler.cpp

```diff
@@ -154,6 +154,7 @@ void AP_InertialSensor::BatchSampler::rotate_to_next_sensor()
             if (_sensor_mask & (1U<<i)) {
                 instance = i;
                 haveinstance = true;
+                post_filter = !post_filter;
```

Comment:
this seems to be not conditional on options for if you've enabled pre+post? what is this bool doing in that case

Reply:
It's ignored if you are not using the option. I could put in an if to only change it if you are using the option but this is more code for no benefit.